test/unit/AccountPageModal.test.js

```diff
@@ -0,0 +1,245 @@
+/* eslint-env mocha */
+/* eslint-disable react/jsx-filename-extension */
+/* global window */
+import React from 'react';
+
+import { expect } from 'chai';
+import { mount } from 'enzyme';
+import sinon from 'sinon';
+import {
+  Button,
+  Card,
+} from '@nypl/design-system-react-components';
+
+import AccountPageModal from './../../src/app/components/AccountPageModal/AccountPageModal';
+
+describe('AccountPageModal', () => {
+  describe('When more than 3 minutes remain', () => {
+    const callRecord = [];
+    let component;
+    let sandbox;
+    let clock;
+
+    before(() => {
+      sandbox = sinon.sandbox.create();
+      clock = sinon.useFakeTimers();
+
+      window.location = { replace: () => {} };
+      global.document.cookie = 'accountPageExp=1615496302599';
+      sandbox.stub(window.location, 'replace').callsFake(() => { callRecord.push({ replace: true }); });
+      sandbox.stub(global, 'setTimeout').callsFake((fn, timeout) => {
+        callRecord.push({ setTimeout: timeout });
+        fn();
+      });
+      sandbox.stub(React, 'useState').callsFake((arg) => {
+        global.update = arg;
```

Comment:
Why the use of `global` here? could you use a variable `update`?

Reply:
I've made this `global` to ensure that the component uses this value when it executes the code below:

```
setTimeout(() => {
      setUpdate(!update);
    }, 1000);
```

which triggers the actual update.